README.md

```diff
@@ -322,6 +322,62 @@ Class['apt::update'] ->
 Class['::mysql::client']
 ```
 
+### Install MySQL 5.6 server on CentOS
+
+This minimal example shows how to install MySQL Community Server 5.6 on Centos 7.3 using Puppet 3.8.7 using hiera with puppetlabs-mysql=3.9.0
+
+Puppet:
+
+```puppet
+  include ::mysql::server
+
+  create_resources(yumrepo, hiera('yumrepo', {}))
+
+  Yumrepo['repo.mysql.com'] -> Anchor['mysql::server::start']
+  Yumrepo['repo.mysql.com'] -> Package['mysql_client']
+
+  create_resources(mysql::db, hiera('mysql::server::db', {}))
+```
+
+Hiera entry:
+
+```yaml
+---
+# mysql module requies feeding it a bunch of parameters to properly install mysql, and not mariadb
```

Comment:
typo in "requires"

suggest changing "...and not mariadb" to "instead of mariadb" for clarity